src/components/pagination/table-pagination.tsx

```diff
@@ -29,27 +38,30 @@ const defaultOptions = [
   { label: '48', value: 48 },
 ];
 
-const additionStyles = () => ({
-  valueContainer: () => ({
+const customSelectStyles = (styles) => ({
+  valueContainer: (p, s) => ({
```

Comment:
what is p and s?

Reply:
provided styles, and select's state. will rename this